Rakefile

```diff
@@ -1,6 +1,37 @@
 require 'bundler'
 require 'bundler/gem_tasks'
+require 'bump'
+require 'github_changelog_generator'
 
 task :spec do
   require File.expand_path('spec/run')
 end
+
+desc 'Sanity Checks and releases gem to rubygems'
+task :sanity do
```

Comment:
`publish` or `push` makes more sense to me. Sanity checks should not actually modify anything - they should be a gate.